server-protocol.md

```diff
@@ -72,19 +72,18 @@ messages. Clients must ignore unrecognized message types from the Server.
 
 The first thing the server sends to each client is the `welcome` message.
 This is intended to deliver important status information to the client that
-might influence its operation. The Python client currently reacts to the
-following keys (and ignores all others):
+might influence its operation. Clients should look out for the following fields,
+and handle them accordingly, if present:
```

Comment:
Maybe add "clients should simply ignore other fields they don't understand"...?

Reply:
I thought this was kind of implicit somehow, but no harm done in stating it explicitly.

Reply:
:+1: yeah, "explicit usually better" IME.